zuul.d/jobs.yaml

```diff
@@ -454,6 +463,20 @@
     vars:
       os_sdk: openstacksdk
 
+- job:
+    name: openstacksdk-ansible-functional-devstack-stein
+    parent: init-test
+    description: |
+      Run openstacksdk ansible functional tests against a stein devstack using git stable-2.7 branch version of ansible.
```

Comment:
please put this description into two lines.

Reply:
I corrected this last time when I found the description only show the first line content on the job web page(http://status.openlabtesting.org/jobs), like the base job https://github.com/theopenlab/openlab-zuul-jobs/blob/master/zuul.d/jobs.yaml#L7, please check this, or I can open an issue to trace this bug.

Reply:
we can land this firstly, maybe there is a fix in zuul in future